virt/virt-2-4-release-notes.adoc

```diff
@@ -83,13 +84,17 @@ the resource limits applied to CDI worker Pods.
 * The `virtctl` tool can now use a `DataVolume` when uploading virtual machine disks to the cluster. This helps prevent virtual machines
 from being inadvertently started before an upload has completed.
 
-//CNV-5421 - DataVolumes have been enhanced with conditions that make it easier to understand the state of VM disk import, clone, and upload.
+//CNV-5421 - OpenShift Container Storage (OCS) DataVolumes 
+* OpenShift Container Storage (OCS) DataVolumes have been enhanced with conditions and events that make it easier to understand the state of virtual disk imports, clones, and upload operations.
+Conditions and events also simplify troubleshooting.
 
 [id="virt-2-4-web-new"]
 === Web console
 //Placeholder for new content.
 
 //CNV-5275 - Update template designs - virtual machines and virtual machine templates merged under one menu called Virtualization.
+* On the Web console, the two sidebar menu items *Virtual Machines* and *Virtual Machine Templates* are replaced by a
```

Comment:
suggestion: s/the two sidebar menu items/the sidebar menu items labeled

Reply:
Done.